shell/browser/api/electron_api_app_mac.mm

```diff
@@ -58,6 +59,16 @@
   [NSApp setActivationPolicy:activation_policy];
 }
 
+bool App::IsRunningUnderRosettaTranslation() const {
+  int proc_translated = 0;
```

Comment:
Let's use Chrome's: https://source.chromium.org/chromium/chromium/src/+/master:base/mac/rosetta.mm;l=32;drc=ef4d087f2a9cbcc2c78c93dccc02695e4c2ed4ba

Reply:
@nornagon Chromes isn't available in 11-x-y.  I'll land this, clean backport and then refactor to use Chromes rosetta helpers on master.

Reply:
sounds good.